frontend/src/routes/Applications/AdvancedConfiguration.tsx

```diff
@@ -69,69 +153,160 @@ export default function AdvancedConfiguration() {
+                        </Text>
+                    </Text>
+                </TextContent>
+            </AcmExpandableCard>
+        )
+    }
+
+    function getSelectedId(props: ISelectedIds) {
+        let { options, query, queryParam, defaultOption, location } = props
+        if (!query) {
+            query = queryString.parse(location.search)
+        }
+        const validOptionIds = options.map((o) => o.id)
+        return queryParam && validOptionIds.includes(query[queryParam]) ? query[queryParam] : defaultOption
+    }
+
+    function QuerySwitcher(props: IQuerySwitcherInterface) {
+        const { options, defaultOption, queryParam = 'resources' } = props
+        const query = queryString.parse(location.search)
+        const selectedId = getSelectedId({
+            query,
+            options,
+            defaultOption,
+            queryParam,
+        })
+
+        let history = useHistory()
+
+        const isSelected = (id: string) => id === selectedId
+        const handleChange = (_, event) => {
+            const id = event.currentTarget.id
+            if (queryParam) {
+                query[queryParam] = id
+            }
+            const newQueryString = queryString.stringify(query)
+            const optionalNewQueryString = newQueryString && `?${newQueryString}`
+            history.replace(`${location.pathname}${optionalNewQueryString}${location.hash}`, { noScrollToTop: true })
+        }
+
+        return (
+            <ToggleGroup>
+                {options.map(({ id, contents }) => (
+                    <ToggleGroupItem
+                        key={id}
+                        buttonId={id}
+                        isSelected={isSelected(id)}
+                        onChange={handleChange}
+                        text={contents}
+                    />
+                ))}
+            </ToggleGroup>
+        )
+    }
+
+    function AdvancedConfigurationTable() {
+        const defaultOption = 'subscriptions'
+        const options = [{ id: 'subscriptions' }, { id: 'channels' }, { id: 'placements' }, { id: 'placementrules' }]
+
+        const selectedId = getSelectedId({ location, options, defaultOption, queryParam: 'resources' })
+        const selectedResources = _.get(table, selectedId)
+
+        return (
+            <AcmTablePaginationContextProvider localStorageKey="advanced-tables-pagination">
+                <AcmTable<IResource>
+                    plural={t('Applications')}
+                    columns={selectedResources.columns}
+                    keyFn={keyFn}
+                    items={selectedResources.items}
+                    extraToolbarControls={
+                        <QuerySwitcher
+                            key="switcher"
+                            options={options.map(({ id }) => ({
+                                id,
+                                contents: t(`application:resource.${id}`),
```

Comment:
We would need the id to determine which toogle is selected and render the corresponding title. Thus I would need useTranslation('application'). @RobDolares would this affect your translation work that you commit in https://github.com/open-cluster-management/console/pull/986 ?

Reply:
If so, would you recommend any alternative ways of finding the title?

Reply:
Can you add the title to your metadata above and use `selectedResources.title`?

Reply:
Sure that's def doable. Thanks Kevin 👍